includes/class-wc-privacy.php

```diff
@@ -369,8 +369,58 @@ protected static function delete_inactive_accounts_query( $timestamp, $limit = 2
 				require_once ABSPATH . 'wp-admin/includes/user.php';
 			}
 
+			$builtin_post_types = get_post_types( array( '_builtin' => true ) );
+			unset( $builtin_post_types['user_request'] ); // Exclude user_request from data migration.
+
+			/**
+			 * Filter post types that should be assign to an admin user prior user exclusion.
+			 *
+			 * @param array $post_types List of post types to assign to an admin. Defaults to WordPress builtin post types.
+			 */
+			$post_types_to_assign = apply_filters( 'woocommerce_delete_inactive_account_post_types_to_assign', array_values( $builtin_post_types ) );
+
 			foreach ( $user_ids as $user_id ) {
-				wp_delete_user( $user_id );
+				$posts_made_prior = get_posts(
```

Comment:
`get_posts` defaults to returning 5 posts - https://developer.wordpress.org/reference/functions/get_posts/. To return all post objects by the user `numberposts` needs to be `-1`.

Reply:
Done! =)